keyboards/gray_studio/space65/keymaps/keithlo/keymap.c

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2019 MechMerlin
```

Comment:
You should probably change the copyright here :)

Reply:
will do :)